recipe/meta.yaml

```diff
@@ -18,6 +18,13 @@ source:
 
 build:
   number: 1
+  noarch: python
+
+requirements:
+  host:
+    - {{ min_python }}
+  run:
+    - {{ min_python }}
```

Comment:
Why do we need this? The CI ran though before adding this - this would only be needed if the `PYTHON` hack does not exist

Reply:
last i checked, it impacts the solved hash.

Reply:
I don't see how leaving this out can impact solving negatively. Putting it in does not change the requirements of any output since they are all overwritten.

Reply:
So as long nothing breaks, let's keep it without